www/src/js/templates/branding-header.hbs

```diff
@@ -0,0 +1,30 @@
+<div class="eight-col last-col">
```

Comment:
Feels like the some hero device icon or image should appear somewhere as well. Check the design mockups and look for missing assets maybe.